lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -130,6 +131,36 @@ def add_pods_library
           end
         end
 
+        # Add a reference to pods' resources (like xcassets) to the user target
+        #
+        # @return [void]
+        #
+        def add_pods_resources
+          UI.puts "- User targets being integrated: #{native_targets.map(&:name).inspect}"
```

Comment:
Yeah I know but those `UI.puts` were only there for debugging for now :laughing: (I'm surprised that I didn't even use `puts` directly for that quick & dirty debug ^^). Proper output with `UI.labeled` will be done later (4th checkbox)